di_website/spotlight/models.py

```diff
@@ -18,13 +22,23 @@ class Meta():
     country_code = models.CharField(max_length=100, help_text='e.g. UG, KE', default='')
     country_name = models.CharField(max_length=255)
     currency_code = models.CharField(max_length=100, help_text='UGX, KES', default='')
-
+    datasources_description = models.TextField(help_text='A description for data sources section', default='')
+    datasources_links = StreamField(
+        DatasourcesStreamBlock,
+        verbose_name="Datasources Links",
+        null=True,
+        blank=True
+    )
     content_panels = Page.content_panels + [
         MultiFieldPanel([
             FieldPanel('country_code'),
             FieldPanel('country_name'),
             FieldPanel('currency_code')
-        ], heading='Settings')
+        ], heading='Settings'),
+        MultiFieldPanel([
+            FieldPanel('datasources_description'),
+            StreamFieldPanel('datasources_links')
+        ], heading='Information for Datasources Section')
```

Comment:
More than just information, right? `Data Sources Section` is fine.

Reply:
Yup, now reads just `Data Sources Section`